src/main/java/org/mastodon/mamut/collaboration/settings/MastodonGitSettingsService.java

```diff
@@ -3,16 +3,29 @@
 import net.imagej.ImageJService;
 
 import org.eclipse.jgit.lib.PersonIdent;
-import org.scijava.service.Service;
 
 public interface MastodonGitSettingsService extends ImageJService
 {
 	boolean isAuthorSpecified();
 
+	/**
+	 * May show a dialog to ask the user for the author name.
+	 */
+	void askForAuthorName();
+
 	void setAuthorName( String name );
```

Comment:
This method is not used. Can it be deleted?

Reply:
The class `MastodonGitController` uses this method. But there are other unused method getAuthorName etc. which I will still keep.